client/my-sites/media-library/test/content.jsx

```diff
@@ -20,31 +25,25 @@ const googleConnectionInvalid = {
 
 const mediaValidationErrorTypes = [ ValidationErrors.SERVICE_AUTH_FAILED ];
 
-function getMediaContent( props ) {
-	return shallow(
-		<MediaLibraryContent
-			mediaValidationErrorTypes={ [] }
-			translate={ noop }
-			site={ { ID: 1 } }
-			{ ...props }
-		/>
-	);
-}
-
-function getMediaContentInstance( props ) {
-	return getMediaContent( props ).instance();
-}
+const defaultProps = {
+	filterRequiresUpgrade: false,
+	mediaScale: 1,
+	mediaValidationErrorTypes: [],
+	selectedItems: [],
+	site: { ID: 1 },
```

Comment:
Wanted to point out that I'm always scared to use `siteId: 1` anywhere, even in tests 😅 Perhaps we could make it a more random number.